content/talks/dynamic-css.md

```diff
@@ -56,7 +56,6 @@ events:
     slides: https://talks.oddbird.net/dynamic-css/vueconf19/
     video: https://www.vuemastery.com/conferences/vueconf-us-2019/dynamic-css-with-vue/
   - venue: Front Range Front-End
-    url: https://www.meetup.com/front-range-front-end/events/chtxtpyxpbcb/
```

Comment:
@mirisuzanne I guess removing these broken links means they now link to the page this is already on? Not a big deal. 🤷

Reply:
@jgerigmeyer works for me